tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1259,19 +1306,12 @@ def func_scalar(self, xp, a, scalar):
     ((2, 3, 4), (2, 0, 1)),
     ((2, 3, 4), (-2, 2, 0)),
 ])
-@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
-def test_sum(is_module, xp, device, shape, axis, keepdims, dtype):
-    a = array_utils.create_dummy_ndarray(xp, shape, dtype)
-    if is_module:
-        out = xp.sum(a, axis=axis, keepdims=keepdims)
-    else:
-        out = a.sum(axis=axis, keepdims=keepdims)
+class TestSum(UnaryMathTestBase, op_utils.NumpyOpTest):
```

Comment:
How about keeping the `is_module` parameterization?

Reply:
Fixed.